docs/reference/cluster/delete-desired-nodes.asciidoc

```diff
@@ -0,0 +1,68 @@
+[[delete-desired-nodes]]
+=== Delete desired nodes API
+++++
+<titleabbrev>Delete desired nodes</titleabbrev>
+++++
+
+NOTE: {cloud-only}
+
+Delete desired nodes.
+
+[[delete-desired-nodes-request]]
+==== {api-request-title}
+
+//////////////////////////
+[source,console]
+--------------------------------------------------
+PUT /_internal/desired_nodes/history/1
+{
+    "nodes" : [
+        {
+            "settings" : {
+                 "node.name" : "instance-000187",
+                 "node.external_id": "instance-000187",
+                 "node.roles" : ["data_hot"],
+                 "node.attr.data" : "hot",
+                 "node.attr.logical_availability_zone" : "zone-0"
+            },
+            "processors" : 8,
+            "memory" : "58gb",
+            "storage" : "1700gb",
+            "node_version" : "8.1.0"
```

Comment:
I think there isn't a way to fetch this value programatically and populate it, I think we should upgrade it as new versions come out. Maybe it's better to skip the doc tests?

Reply:
I think it is ok to leave that for a follow-up at least.